sqlserver/assets/configuration/spec.yaml

```diff
@@ -190,6 +190,13 @@ files:
         type: array
         items:
           type: string
+    - name: include_db_file_space_usage_metrics
+      description: |
+        Include database file space usage metrics for how space is used by data files in a specific database. 
```

Comment:
`Include tempdb file space usage metrics for how space is used in tempdb data files.`